app/src/main/java/com/github/se/wanderpals/ui/screens/trip/agenda/Agenda.kt

```diff
@@ -174,17 +179,34 @@ fun Banner(agendaViewModel: AgendaViewModel, isExpanded: Boolean, onToggle: () -
       modifier =
           Modifier.fillMaxWidth()
               .clickable { onToggle() }
-              .padding(16.dp)
-              .background(primaryLight)
+              .padding(8.dp)
+              .background(MaterialTheme.colorScheme.primary)
               .testTag("Banner")) {
-        DisplayDate(date = selectedDate)
-        // Optional: Add an icon to indicate the expand/collapse action
-        Icon(
-            imageVector =
-                if (isExpanded) Icons.Default.KeyboardArrowUp else Icons.Default.KeyboardArrowDown,
-            contentDescription = "Toggle",
-            modifier = Modifier.align(Alignment.CenterEnd),
-            tint = Color.White)
+        Row(modifier = Modifier.fillMaxWidth(), verticalAlignment = Alignment.CenterVertically) {
+          DisplayDate(date = selectedDate, color = MaterialTheme.colorScheme.onPrimary)
+          // Optional: Add an icon to indicate the expand/collapse action
+          Icon(
+              imageVector =
+                  if (isExpanded) Icons.Default.KeyboardArrowUp
+                  else Icons.Default.KeyboardArrowDown,
+              contentDescription = "Toggle",
+              tint = MaterialTheme.colorScheme.onPrimary)
+          Spacer(modifier = Modifier.weight(1f))
+          // Add an icon to tap that opens the full list of all stops for the trip
+          IconButton(
+              onClick = {
+                // Navigate to the stops list screen
+                navigationActions.navigateTo(Route.STOPS_LIST)
```

Comment:
When clicking on this icon button, we navigate to the display-stop-list screen 👍🏻